pages/linux/sfill.md

```diff
@@ -0,0 +1,20 @@
+# sfill
+
+> Secure overwrite free space and inodes of the partition where the specified directory resides.
+> More information: <https://manned.org/sfill>.
+
+- Overwrite free space and inodes of an USB with 38 writes (slow but secure):
```

Comment:
Is `38` customizable here?

Reply:
By default:
```bash
       The secure data deletion process of sfill goes like this:

       *      1 pass with 0xff

       *      5 random passes. /dev/urandom is used for a secure RNG if
              available.

       *      27 passes with special values defined by Peter Gutmann.

       *      5 random passes. /dev/urandom is used for a secure RNG if
              available.
```

`1+5+27+5 = 38`

When using the `-l` flag:
```bash
       -l     lessens the security. Only two passes are written: one mode with
              0xff and a final mode with random values.
```
So that makes `6`.

Reply:
Correct, 38 is default and not strictly customizable. I [fixed](545bd8b7f9dc6b33680e91426fea1a7fe917fd7a) the number of writes with '-l' option and added  a new example for the '-ll' mode.